### PR TITLE
Adapter  screen-adapter on Taobao

### PR DESCRIPTION
**Pull request by yyycggg**
Re: https://github.com/cocos/3d-tasks/issues/13998

### Changelog

* Adapter  screen-adapter on Taobao

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Review by PPpro — APPROVED** [review]

File: pal/minigame/taobao.ts [hunk 1]
```diff
@@ -93,16 +93,17 @@ minigame.stopAccelerometer = function (res: any) {
 // #endregion Accelerometer
 
 // #region SafeArea
+// It should be a value that is not multiplied by dpr
 minigame.getSafeArea = function () {
```
Comment (minggo) [inline pal/minigame/taobao.ts:97]: Why not multiply by dpr. As @PPpro said, the app only care about pixel size.

File: platforms/minigame/platforms/taobao/wrapper/unify.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 const utils = require('./utils');
 
 function handleTouchEvent (event) {
-	if (my.isIDE) {
-		return;
-	}
+	// if (my.isIDE) {
+	// 	return;
+	// }
```
Comment (minggo) [inline platforms/minigame/platforms/taobao/wrapper/unify.js]: Please remove it if it is not needed.

File: pal/minigame/taobao.ts (hunk 1 above)
Comment (yyycggg) [reply pal/minigame/taobao.ts:97]: this function should return a value that is not multiplied by pdr, beacuse `PAL` will multiply by dpr. We will fix it later.

File: platforms/minigame/platforms/taobao/wrapper/unify.js (hunk 2 above)
Comment (yyycggg) [reply platforms/minigame/platforms/taobao/wrapper/unify.js]: OK.

**Review by minggo — APPROVED** [review]

**Pull request merged**